### ISSUE TITLE
Avoid breaking words

### ISSUE THREAD

**Issue by rsimmons-eca** [anovis/autosizetext#11]
![Screenshot 2024-11-18 at 15 03 01](https://github.com/user-attachments/assets/51f4abda-2877-45ab-92d0-e2477c5490a8)

It would be good if we could avoid breaking words somehow. Like resize text up till the point where a word is broken across lines

**Comment by anovis** [comment]
@rachelcsimmons-cda yeah that makes sense! ill try and see what is possible with detecting that and shuffling words around.
